test/UnitTests/CheckoutControllerTest.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Xunit;
+using MusicStore.Models;
+
+namespace MusicStore.Controllers 
+{
+    public class CheckoutControllerTest
+    {
+        private readonly IServiceProvider _serviceProvider;
+
+        public CheckoutControllerTest()
+        {
+            var services = new ServiceCollection();
+
+            services.AddEntityFramework()
+                      .AddInMemoryStore()
+                      .AddDbContext<MusicStoreContext>();
+
+            _serviceProvider = services.BuildServiceProvider();
+        }
+
+        [Fact]
+        public void AddressAndPayment_ReturnsDefaultView()
+        {
+            // Arrange
+            var controller = new CheckoutController();
+
+            // Act
+            var result = controller.AddressAndPayment();
+
+            // Assert
+            var viewResult = Assert.IsType<ViewResult>(result);
```

Comment:
Is this assert needed? Doesn't the type *have* to be a ViewResult for this to even compile? (In particular, the next assert)